api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -194,7 +194,7 @@ type VolumeReplicationGroupSpec struct {
 type Identifier struct {
 	// ID contains the globally unique storage identifier that identifies
 	// the storage or replication backend
-	ID string `json:"storageID"`
+	ID string `json:"id"`
```

Comment:
should this be storageid?

Reply:
No, this was copied from an original StorageID field, this is just id, so changed that as well.